helper-scripts/sort-versions.js

```diff
@@ -0,0 +1,45 @@
+/* eslint-env node */
+
+const fs = require('fs');
+
+const fileName = process.argv.slice(2)[0];
+
+console.log(`Sorting version arrays in ${fileName}`);
+
+const data = fs.readFileSync(fileName);
+const compatData = JSON.parse(data.toString());
+
+const apiName = Object.keys(compatData.api);
+const methods = Object.keys(compatData.api[apiName]);
+
+const api = compatData.api[apiName];
+
+methods.forEach(method => {
```

Comment:
Nit: Actually these are `features`, not necessarily only methods.

Reply:
Also you could replace the assignment to `const methods` by `Object.keys(api).forEach(feature => {` here.

Reply:
Git